day_2/project_of_the_day.py

```diff
@@ -0,0 +1,18 @@
+# https://replit.com/@appbrewery/tip-calculator-end
+
+GREETING = 'Welcome to the tip calculator!'
+ASK_TOTAL_BILL = 'What was the total bill?'
+ASK_TIP_PERCENT = 'How much tip would you like to give? 10, 12, or 15?'
```

Comment:
It looks like options that I should choose from, but it's not, maybe use different text, or explain the limitations (e.g. only positive integer, less than 100)

Reply:
Actually I tried to reproduce the program that is pinned at the top, so I didn`t change the structure.

Reply:
I think I`ll make limitations so that only the listed values are acceptable.

Reply:
I think I`ll make limitations so that only the listed values are acceptable.

Reply:
I think I`ll make limitations so that only the listed values are acceptable.

Reply:
bup

Reply:
bup`